modules/star/align/main.nf

```diff
@@ -67,6 +66,8 @@ process STAR_ALIGN {
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         star: \$(STAR --version | sed -e "s/STAR_//g")
+        samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//')
```

Comment:
`samtools` is in the container but not used in the script section, should we report its version? won't it be misleading?`

Reply:
Well spotted. You are right. I thought it made sense to use the same container as `STAR_GENOMEGENERATE` to save having to download an additional container just for STAR. Don't think there is any harm in reporting the version of SAMtools if it's in the container?

Reply:
My concern is that if a user checks the versions reported by the module in the multiqc report, they would expect that `samtools` has been used in the process and it will be hard to spot that the reason is just that we are using the same container (this I agree that makes sense). Maybe I am over paranoid :laughing:

Reply:
Yep, but if we use a mulled container here and say someone updates just SAMtools we won't know if we don't report it explicitly.